app/routes/helpers/claim-summary-helper.js

```diff
@@ -51,6 +51,10 @@ module.exports.getDocumentFilePath = function (claimDocumentId) {
           name: `${DEFAULT_FILE_NAME}.${path.split('.').pop()}`
         }
       }
+
```

Comment:
This test and the code were just wrong. It had no catch block so I don't understand the test ever passed.